.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "printWidth": 120,
+  "semi": false,
```

Comment:
How tied are we to this `prettierrc`? Would be nice to use the defaults as much as possible unless people have any strong opinions otherwise?